.env

```diff
@@ -1,2 +1,2 @@
-NODE_TAG=v6.4.0
+NODE_TAG=v0.0.0-hc5
```

Comment:
is this temporary to run the tests ?

Reply:
I'd stick to official node releases instead.